src/Symfony/Component/Debug/Tests/ErrorHandlerTest.php

```diff
@@ -144,7 +144,9 @@ public function testNotice()
         restore_error_handler();
     }
 
-    // dummy function to test trace in error handler.
+    /**
+     * Dummy function to test trace in error handler.
+     */
```

Comment:
Not sure which way to go. I think I would prefer to keep the current code which is perfectly valid.